.bumpversion.cfg

```diff
@@ -3,9 +3,13 @@ current_version = 0.3.0
 commit = True
 tag = True
 
-[bumpversion:file:setup.cfg]
-search = version = {current_version}
-replace = version = {new_version}
+# [bumpversion:file:setup.cfg]
```

Comment:
If the setup.cfg has been deleted these lines can be removed rather than just commented out